Sources/silt/main.swift

```diff
@@ -21,13 +21,16 @@ import CommandLineKit
 func parseOptions() -> (Options, Set<String>) {
   let cli = CommandLineKit.CommandLine()
   let dumpOption =
-    EnumOption<Mode.DumpKind>(longFlag: "dump", required: false,
-      helpMessage: "Dumps the compiler's input at the specified stage in the " +
-                   "compiler.")
+  EnumOption<Mode.DumpKind>(longFlag: "dump", required: false,
+    helpMessage: "Dumps the compiler's input at the specified stage in the " +
+                 "compiler.")
+  let parseVerify =
+    BoolOption(longFlag: "parse-verify",
```

Comment:
Nit: Could this just be called `verify`?

Reply:
I possibly want to add a verify step for different layers -- maybe this could be `--verify parse`?